roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -86,7 +86,7 @@ kube_script_dir: "{{ bin_dir }}/kubernetes-scripts"
 kube_manifest_dir: "{{ kube_config_dir }}/manifests"
 
 # This is where all the cert scripts and certs will be located
-kube_cert_dir: "{{ kube_config_dir }}/ssl"
+kube_cert_dir: "{{ kube_config_dir }}/pki"
```

Comment:
This will break upgrades from non-kubeadm and other existing clusters unless you also add migration from the old cert path to the new.

Reply:
yes it will brake if use only master branch, not sure if its even possible to create migration without reconfiguring all static pods, that is not good at all. There is another workaround:
1) add notice that this is old configuration that need to be remove because kubernetes not use this as default
2) keep example of inventory with ssl folder not pki

Not sure if its better but i think if we will support all possible solution it will be impossible to add something without braking other

Reply:
change folder back to ssl